contracts/SecurityTokenRegistry.sol

```diff
@@ -532,18 +554,20 @@ contract SecurityTokenRegistry is EternalStorage, Proxy {
      * @param _securityToken is the address of the securityToken
      * @param _tokenDetails is the off-chain details of the token
      * @param _deployedAt is the timestamp at which the security token is deployed
+     * @param _protocolVersion Version of securityToken contract
      */
     function modifySecurityToken(
         string calldata _name,
         string calldata _ticker,
         address _owner,
         address _securityToken,
         string calldata _tokenDetails,
-        uint256 _deployedAt
+        uint256 _deployedAt,
+        uint256 _protocolVersion
```

Comment:
Do you think this is needed for the modifyST? It may be sufficient to just set this to 0 in the event. This data would only be as good as the value passed in, and in practice we'll care more about what the deployed version of the token (from the token itself) is, rather than parsing these events.

Reply:
Yes, I was also thinking the same. But I did it because of consistency. I will make the change anyway.